client/src/components/productList/OrderFilter/index.style.ts

```diff
@@ -0,0 +1,30 @@
+import styled from 'styled-components';
+
+export const OrderFilterWrapper = styled.div`
+  img {
+    cursor: pointer;
+  }
+`;
+
+export const OrderHeder = styled.div`
+  font-size: 25px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 10px;
+  margin-bottom: 35px;
+  img {
+    width: 25px;
+    &:hover {
+      transition: all 300ms;
```

Comment:
transform만 걸어도 될것 같아요~ 혹시 다른 것도 걸려있는게 있을까요?

```suggestion
      transition: transform 300ms;
```

Reply:
고부분은 순서 컴포넌트 리셋 버튼에 대한 트랜지션입니다

Reply:
수정완료!